openapi/customervoice/customervoice.yaml

```diff
@@ -231,8 +500,121 @@ components:
                       type: string
                       default: template
                       readOnly: true
+    templates:
+      title: Templates
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+
+        Templates that can be used for emails and other messaging services.
+      type: object
+      x-examples:
+        Example Template:
+          type: templates
+          id: 'AG-abc123:email:TEM-d2f4cf1aa7ae480bbf0a2da67dxxxxxx'
+          attributes:
+            name: My Preferred Sites Review Request
+            content: |-
+              Hi {{.customerName}},
+              <br>
+              <br>
+              Thank you for your business! We'd love to know more about your latest experience with us.
+              <br>
+              <br>
+              {{.buttonUrl}}
+              <br>
+              <br>
+              Thank you,
+              <br>
+              John Doe
+            metadata:
+              subject: Thank you for choosing us!
+              replyEmail: jondoe@mail.com
+            locale: en-US
+            type: email
+          relationships:
+            businessLocation:
+              data:
+                id: AG-12345
+      x-lifecycle:
+        status: trustedTester
+      x-tags:
+        - Templates
+      properties:
+        id:
+          type: string
+          example: AG-abc123:email:TEM-d2f4cf1aa7ae480bbf0a2da67dxxxxxx
+          readOnly: true
+        type:
+          type: string
+          default: templates
+          enum:
+            - templates
+          readOnly: true
+        attributes:
+          type: object
+          properties:
+            name:
+              type: string
+              description: A user friendly description for the template.
+              example: My Preferred Sites Review Request
+            metadata:
+              type: object
+              description: A JSON object containing key-value pairs that can be used to store additional information about the template.
+              example:
+                subject: Thank you for choosing us!
+                replyEmail: jondoe@mail.com
+            content:
+              type: string
+              description: The content of the template.
+              example: |-
+                Hi {{.customerName}},
+                <br>
+                <br>
+                Thank you for your business! We'd love to know more about your latest experience with us.
+                <br>
+                <br>
+                {{.buttonUrl}}
+                <br>
+                <br>
+                Thank you,
+                <br>
+                John Doe
+            locale:
+              type: string
+              description: The locale of the template.
+              example: en-US
```

Comment:
It's too bad that CV doesn't autoselect a subtemplate based on the customer's preferred language. When I look at the CV UI it appears that this field is selecting which translation to use for the email footer. Only English and French are options. 

BTW We used the following for the `platform/users` resource to indicate the user's preferred language. 

```yaml
            languageLocaleCode:
              type: string
              pattern: '^[a-z]{2}-[A-Z]{2}$'
              example: en-US
              description: |-
                The local language that the template was written in.

                Note: We currently have a limited set of allowed languages but expect to support many more in the future. When the requested language is not supported US English will be used.

                It should be a single BCP47/RFC5646 language tag. This is an [ISO 639-1 Alpha-2](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) language code in lowercase and an [ISO 3166-1 Alpha-2](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes) country code in uppercase, separated by a dash.

                Examples:
                - `en-US` English (US)
                - `fr-FR` French (France)
                - `cs-Cz` Czech (Czech Republic)
                - `de-DE` German (Germany)
                - `nl-BE` Dutch (Belgium)
```